instrumentor/controllers/startlangdetection/manager.go

```diff
@@ -83,5 +83,19 @@ func SetupWithManager(mgr ctrl.Manager) error {
 		return err
 	}
 
+	err = builder.
```

Comment:
this webhook is used by startlangdetection and deleteinstrumentationconfig controllers, so it can be set on the main instrumentor manager like it originally was